charts/alfresco-connector-ms365/templates/deployment-connector-ms365.yaml

```
@@ -62,17 +62,25 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- include "component-security-context" .Values | indent 8 }}
-          envFrom:
-            - configMapRef:
-                name: {{ .Values.repository.existingConfigMap | default (print (include "alfresco-connector-ms365.fullname" .)) }}
+          {{- $repo_details_cm := .Values.repository.existingConfigMap.name | default (include "alfresco-connector-ms365.fullname" .) }}
+          {{- $repo_url := printf "-Dalfresco.base-url=http://%s:%s" "$REPO_HOST" "$REPO_PORT" }}
           env:
             {{- range $key, $value := (omit .Values.environment "JAVA_OPTS") }}
             - name: {{ $key }}
               value: {{ $value }}
             {{- end }}
             - name: JAVA_OPTS
-              value: >-
-                {{ (printf "%s %s" (.Values.environment.JAVA_OPTS | default "") "-Dalfresco.base-url=$REPO_URL") }}
+              value: {{ printf "%s%s" (.Values.environment.JAVA_OPTS | default "") ($repo_url) }}
```

without a space you may mess up the java command line

```suggestion
              value: {{ printf "%s %s" (.Values.environment.JAVA_OPTS | default "") ($repo_url) }}
```